File: C++/FizzBuzz.cpp
// FizzBuzz Problem using C++
// Author: @gourab337

#include<bits/stdc++.h>

using namespace std;

int main(){

 for (int i=1; i<101; i++){
     if(i%3 == 0)
         cout<<"FIZZ ";
     if(i%5 == 0)
      cout<<"BUZZ ";
     if((i%3)*(i%5))    // if (i%3) && (i%5) !=0, then only the if case gets executed
        cout<<i<<" ";
     
     cout<<",";
}

}

File: Haskell/fizzBuzz.hs
--author : @mainak-debnath
module Main where
main :: IO ()
main = printAll $ map fizzBuzz [1..100]
 where
 printAll [] = return ()
 printAll (x:xs) = putStrLn x >> printAll xs
fizzBuzz :: Integer -> String
fizzBuzz n | n `mod` 15 == 0 = "FizzBuzz"
 | n `mod` 5 == 0 = "Fizz"
 | n `mod` 3 == 0 = "Buzz"
 | otherwise = show n